docs/process/release.md

```diff
@@ -10,7 +10,17 @@
 
 The caretaker should triage issues, merge PR, and sheppard the release.
 
-Caretaker calendar can be found [here](https://calendar.google.com/calendar?cid=Z29vZ2xlLmNvbV9zZjlvODF0NGE4NzE5ZmtiMnBoZnA4MGk2Z0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
+Caretaker calendar can be found [here](https://calendar.google.com/calendar/embed?src=google.com_j1o56jq9fp68nb6v7t1tvu3af0@group.calendar.google.com).
+
+Each shift consists of two caretakers. The primary caretaker is responsible for
+merging PRs to master and patch whereas the secondary caretaker is responsible
+for the release. Primary-secondary pairs are as follows:
+
+Primary | Secondary
```

Comment:
The calendar link lists the Googlers as primary, I'm wondering if we should switch these roles just to be more consistent? But then having the secondary responsible for merging PRs might also be confusing, so IDK; up to you.